settings.php

```diff
@@ -105,4 +105,8 @@ function ecs($configoption, $params = null) {
     $intercepttype = new admin_setting_configselect('equella_intercept_files', get_string('interceptfiles', 'equella'), get_string('interceptfilesintro', 'equella'), 0, $choices);
 
     $settings->add($intercepttype);
+
+    $settings->add(new admin_setting_heading('equella_lti_migration', ecs('lti13.migration.title'), ''));
```

Comment:
One other quick comment. Note how the sections above have a comment to highlight which part of the settings they below too (e.g. Drag and drop). Could you please add such a comment for this new section too?

Reply:
Too easy. I will add the comment.